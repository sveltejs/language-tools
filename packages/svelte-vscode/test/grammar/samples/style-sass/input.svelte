<style lang="sass">
    .abc
</style>
