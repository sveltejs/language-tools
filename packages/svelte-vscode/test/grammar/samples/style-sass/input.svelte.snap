><style lang="sass">
#^ source.svelte meta.style.svelte meta.tag.start.svelte punctuation.definition.tag.begin.svelte
# ^^^^^ source.svelte meta.style.svelte meta.tag.start.svelte entity.name.tag.svelte
#      ^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte
#       ^^^^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte meta.attribute.lang.svelte entity.other.attribute-name.svelte
#           ^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte meta.attribute.lang.svelte punctuation.separator.key-value.svelte
#            ^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte punctuation.definition.string.begin.svelte
#             ^^^^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte
#                 ^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte meta.attribute.lang.svelte string.quoted.svelte punctuation.definition.string.end.svelte
#                  ^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.tag.start.svelte punctuation.definition.tag.end.svelte
>    .abc
#^^^^^^^^^ source.svelte meta.style.svelte meta.lang.sass.svelte meta.embedded.block.svelte source.sass
></style>
#^^ source.svelte meta.style.svelte meta.tag.end.svelte punctuation.definition.tag.begin.svelte
#  ^^^^^ source.svelte meta.style.svelte meta.tag.end.svelte entity.name.tag.svelte
#       ^ source.svelte meta.style.svelte meta.tag.end.svelte punctuation.definition.tag.end.svelte
>